test/reducers/dist.js

```diff
@@ -61,20 +59,13 @@ test('reducers/dist/dist_apigateway', (t) => {
     }
   };
   const newState = reducer(initialState, action);
-  t.deepEqual(expectedState, newState);
```

Comment:
I'd prefer to add some junk objects for s3Access and teaLambda just to show they aren't changed by the reducer, but this is fine.